week-2-practise/level-2/3-sort.js

```diff
@@ -3,12 +3,12 @@
 
 var letters = ["a", "n", "c", "e", "z", "f"];
 var letters2 = ["w", "b", "v", "g", "l", "o"];
-var sortedLetters; // ONLY EDIT THIS LINE
+var sortedLetters = (letters.concat(letters2)).sort(); // ONLY EDIT THIS LINE
```

Comment:
Are you sure you need the extra parenthesis? :) I understand why you'd think you would though!